language-support/ts/daml-ledger/index.test.ts

```diff
@@ -162,6 +163,26 @@ describe("streamQuery", () => {
     expect(mockChange).toHaveBeenLastCalledWith([fooCreateEvent(1)])
   });
 
+  test("reconnect on close", () => {
+    const ledger = new Ledger({...mockOptions, reconnectThreshold: 0} );
```

Comment:
Can we please use an actual threshold here, maybe 500 millis, and add a sleep statement in the test?

Reply:
added a 500 millisecond sleep in the tests.